decidim-dev/lib/decidim/dev/test/rspec_support/dynamic_attach.rb

```diff
@@ -23,7 +23,7 @@ def dynamically_attach_file(name, file_location, options = {})
           expect(page).to have_content(filename.first(12)) if front_interface
         end
         all(title_input(front_interface)).last.set(options[:title]) if options.has_key?(:title)
-        click_on(front_interface ? "Next" : "Save") unless options[:keep_modal_open]
+        click_on(front_interface = "Save") unless options[:keep_modal_open]
```

Comment:
Before it was a ternary operator, so we just need to drop this without an assignment 

```suggestion
        click_on("Save") unless options[:keep_modal_open]
```